student-work/cassandradelieto/Building Classes/working_with_classes.py

```diff
@@ -0,0 +1,49 @@
+<<<<<<< HEAD
```

Comment:
These are markers for merge conflicts that you would need to remove if you ever want to run the python file.